pkgs/servers/nosql/redis/default.nix

```diff
@@ -20,19 +20,21 @@ stdenv.mkDerivation rec {
       url = "https://github.com/redis/redis/commit/bfe50a30edff6837897964ac3374c082b0d9e5da.patch";
       sha256 = "sha256-0GMiygbO7LbL1rnuOByOJYE2BKUSI+yy6YH781E2zBw=";
     })
+    # use system jemalloc
+    (fetchpatch {
```

Comment:
nitpick, I think we can use `fetchurl`. See: https://github.com/NixOS/nixpkgs/issues/240141